ocp/local.yaml

```diff
@@ -50,6 +50,7 @@ include:
             familyName: Houndigrade
           availabilityZone: us-east-1a
           autoscalingGroupName:
+          scaleUpClusterSchedule: 5
```

Comment:
Maybe 5 minutes instead of 5 seconds? Spamming a task that takes a few minutes to finish every 5 seconds seems a little too much.